frontends/mit-open/src/pages/FieldPage/FieldSearch.tsx

```diff
@@ -35,24 +64,39 @@ const FACETS_BY_CHANNEL_TYPE: Record<ChannelTypeEnum, string[]> = {
   ],
   [ChannelTypeEnum.Unit]: [
     "free",
-    "certification_type",
     "topic",
+    "certification_type",
     "department",
     "learning_format",
   ],
   [ChannelTypeEnum.Pathway]: [],
 }
 
+const SHOW_PROFESSIONAL_TOGGLE_BY_CHANNEL_TYPE: Record<
+  ChannelTypeEnum,
+  boolean
+> = {
+  [ChannelTypeEnum.Topic]: true,
+  [ChannelTypeEnum.Department]: false,
```

Comment:
Both the issue and the PR description seem to say that department channels should still have the professional toggle. Should this be `true`?

Reply:
Ferdi asked me to remove it in a comment on the pr

Reply:
I see that above now, thanks